dags/surveymonkey_data_import_pipeline.py

```diff
@@ -117,6 +119,37 @@ def surveymonkey_survey_questions_etl(**kwargs):
         )
 
 
+def surveymonkey_survey_answers_etl(**kwargs):
+    data_config = data_config_from_xcom(kwargs)
+    survey_id_list_from_df = get_distinct_values_from_bq(
+        project_name=data_config.project_name,
+        dataset_name=data_config.dataset_name,
+        table_name_source=data_config.survey_list_table_name,
+        column_name=data_config.survey_id_column_name
+    ).values.tolist()
+
+    for survey_id_list in survey_id_list_from_df:
+        survey_id = str(survey_id_list[1])
+        LOGGER.info("answers for survey_id: %s", survey_id)
+        list_of_answers_of_one_survey = get_survey_answer_details(
```

Comment:
I think what threw me off a bit, which may be obvious when thinking more about it, is that this is actually the list of answers for each user. Correct? (Or survey responses)
For me intuitively, an answer is a single answer by a single user to a single question.
A survey response I would think of from a single user, could be a collection of answers or a single answer.
i.e. perhaps the naming could be slightly clearer?

Reply:
It is actually collection of all answers for one survey. maybe I can change the name `list_of_all_answers_for_one_survey` to make it more clear. There is **no user breakdown** for surveys in any part of the code.